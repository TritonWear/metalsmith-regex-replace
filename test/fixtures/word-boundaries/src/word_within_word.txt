If a ninja is already hiding in a roninjamboree, there's no need to assist.