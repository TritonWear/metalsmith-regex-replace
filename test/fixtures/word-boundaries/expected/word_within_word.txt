If a blarg is already hiding in a roninjamboree, there's no need to assist.